core-web/libs/dotcms-scss/angular/dotcms-theme/components/_chip.scss

```diff
@@ -1,18 +1,82 @@
 @use "variables" as *;
+@import "mixins";
 
-p-chip {
-    background-color: $color-palette-gray-300;
+p-chip .p-chip {
+    background-color: $color-accessible-text-blue-bg;
+    color: $color-accessible-text-blue;
     border-radius: $border-radius-md;
-    display: inline-block;
-    padding: 0.25rem $spacing-1;
-    position: relative;
-    width: fit-content;
+    padding: 0 $spacing-1;
+    height: $chip-height-md;
+    gap: $spacing-1;
 
     .p-chip-text {
-        font-size: $font-size-sm;
-        line-height: $spacing-4;
-        color: $color-palette-gray-700;
-        vertical-align: middle;
-        white-space: nowrap;
+        font-size: $font-size-md;
+    }
+
+    .p-chip-icon,
+    .pi-chip-remove-icon {
+        justify-content: center;
+        align-items: center;
+        display: flex;
+
+        line-height: 1;
+        aspect-ratio: 1/1;
+        height: calc($icon-md + 4px);
```

Comment:
Hummm... why?

Reply:
`$icon-sm` is `14px` but in the figma is `16px` square. In the case of `$icon-md` is `16px` but the icon in the Figma is `20px`

Reply:
handle the size of the icons globally